src/libtexture/imagecache.cpp

```diff
@@ -3533,7 +3524,7 @@ ImageCache::create (bool shared)
     }
 
     // Doesn't need a shared cache
-    ImageCacheImpl *ic = new ImageCacheImpl;
+    ImageCacheImpl* ic = aligned_new<ImageCacheImpl>();
```

Comment:
Why does it matter if ImageCacheImpl uses aligned_new?

Reply:
Because it contains an `unordered_map_concurrent` which itself contains cache aligned bins.

If the structure doesn't get aligned properly, those bins could end up straddling cache lines, leading to false sharing.

Its somewhat unfortunate that `operator new` doesn't do this correctly by default, but there is no overload of `new` (before c++17) that accepts the alignment as an argument. So anytime you want to `new` a type with alignment larger than the default platform alignment, you need to use such a wrapper.